src/common/telemetry_event/src/lib.rs

```diff
@@ -77,7 +78,15 @@ pub fn report_event_common(
     if let Some(tracking_id) = TELEMETRY_TRACKING_ID.get() {
         event_tracking_id = tracking_id.to_string();
     } else {
-        tracing::info!("Telemetry tracking_id is not set, event reporting disabled");
+        static LOG_SUPPERSSER: LazyLock<LogSuppresser> =
```

Comment:
Think you made a typo here.

Reply:
You are right. The entire codebase is wrong. 😂

<img width="546" height="1312" alt="image" src="https://github.com/user-attachments/assets/5828969d-10f0-432a-9a24-cc00197486ba" />